# Camera Parameters
cam_model: Pinhole
cam_width: 640
cam_height: 480
cam_fx: 744.043998
cam_fy: 743.372287
cam_cx: 325.387252
cam_cy: 218.535647
cam_d0: -0.314025
cam_d1: 0.116978
cam_d2: 0.000833
cam_d3: -0.001095
image_topic: "/camera_down/image_raw"

# Frame Parameters
odom_frame_id: "fcu"
base_link_frame_id: "uav"
sensor_frame_id: "camera"
publish_tf: true
invert_tf: false

# CV Parameters
min_number_of_features: 20
ransac_threshold: 1
ransac_confidence: 0.99
feature_detector_threshold: 20
feature_tracker_window_size: 21
feature_tracker_max_pyramid_level: 2
feature_tracker_max_iterations: 30
feature_tracker_epsilon: 0.01
feature_tracker_eigen_threshold: 0.0001
visualize: true
visualize_frame_tracking: 10
